tfhe/src/core_crypto/entities/lwe_ciphertext_list.rs

```diff
@@ -154,17 +175,41 @@ impl<Scalar: UnsignedInteger> LweCiphertextListOwned<Scalar> {
     /// output.
     ///
     /// See [`LweCiphertextList::from_container`] for usage.
-    pub fn new(
-        fill_with: Scalar,
+    pub fn from_elem(
         lwe_size: LweSize,
         ciphertext_count: LweCiphertextCount,
         ciphertext_modulus: CiphertextModulus<Scalar>,
+        fill_with: Scalar,
     ) -> Self {
-        Self::from_container(
-            vec![fill_with; lwe_size.0 * ciphertext_count.0],
-            lwe_size,
-            ciphertext_modulus,
-        )
+        let container_size = lwe_size.0 * ciphertext_count.0;
+        let container = vec![fill_with; container_size];
+        Self::from_container(container, lwe_size, ciphertext_modulus)
+    }
+
+    /// Allocate memory and create a new owned [`LweCiphertextList`], where each element
+    /// is provided by the `fill_with` function, invoked for each consecutive index.
+    ///
+    /// # Note
+    ///
+    /// This function allocates a vector of the appropriate size and wraps it in the appropriate
+    /// type. If you want to encrypt data you need to use
+    /// [`crate::core_crypto::algorithms::encrypt_lwe_ciphertext_list`] or its parallel variant
+    /// [`crate::core_crypto::algorithms::par_encrypt_lwe_ciphertext_list`] using this list as
+    /// output.
+    ///
+    /// See [`LweCiphertextList::from_container`] for usage.
```

Comment:
I don't see an update to the from_container docstring to demonstrate usage ?

Reply:
I added an usage example now.